tests/test_cli.py

```diff
@@ -87,6 +87,38 @@ def test_init(app, template_entrypoints):
 
     assert current_search_client.indices.exists(list(search.mappings.keys()))
 
+    # Test option --index-name
+    index = 'records-authorities-authority-v1.0.0'
+    suffixed_index = 'records-authorities-authority-v1.0.0-abc'
+    expected_aliases = [
+        'records',
+        'records-authorities',
+        'records-authorities-authority-v1.0.0'
+    ]
+
+    assert current_search_client.indices.exists(suffixed_index)
+    result = runner.invoke(
+        cmd,
+        ['delete', '--verbose', '--yes-i-know', suffixed_index],
+        obj=script_info
+    )
+    assert 0 == result.exit_code
+    assert not current_search_client.indices.exists(suffixed_index)
+    aliases = current_search_client.indices.get_alias()
+    assert suffixed_index not in list(aliases.keys())
+
+    result = runner.invoke(
+        cmd,
+        ['init', '--index-name', index],
+        obj=script_info
+    )
+    assert 0 == result.exit_code
+    assert current_search_client.indices.exists(list(search.mappings.keys()))
+    aliases = current_search_client.indices.get_alias()
+    assert 8 == sum(len(idx.get('aliases', {})) for idx in aliases.values())
```

Comment:
I had a lot of issues in the past (on `cds-videos`, another Invenio installation) with tests that contains these numbers, like `8 ==`, because when changing something, it is hard to know what number to put there and why such number.

Could you try to replace the number with a named constant or maybe explain in a comment?